src/compile/common.ts

```diff
@@ -103,7 +103,9 @@ export function formatMixins(model: Model, channel: Channel, format: string) {
   let def: any = {};
 
   if (fieldDef.type === TEMPORAL) {
```

Comment:
Yes, if we have a default format. But in that case the formatType should be set automatically.